lifelines/estimation.py

```diff
@@ -1152,6 +1158,10 @@ def _newton_rhaphson(self, X, T, E, initial_beta=None, step_size=1.,
                                    include_likelihood=include_likelihood)
             # Do not override hessian and gradient in case of garbage
             h, g = output[:2]
+            if self.penalizer > 0:
+                # add the gradient and hessian of the l2 term
+                g -= self.penalizer * beta.T
```

Comment:
hey @benkuhn thanks for adding this feature! I'm just looking at the code to try to understand L2 reg in the context of cox fitting, and wanted to know why the reg term is `self.penalizer * beta.T` instead of `self.penalizer * beta.T^2`?